app/src/main/res/layout/fragment_settings_safe.xml

```diff
@@ -1,159 +1,167 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout
+<androidx.coordinatorlayout.widget.CoordinatorLayout
```

Comment:
Same as the previous 2 comments. Is it absolutely necessary to nest? If it is feel free to ignore this comment.